src/lib/generators/code-snippet-generators.ts

```diff
@@ -14,12 +15,23 @@ export const generateInterfaceDefinition = (interfaceName: string, typeName: str
         data: string;}`;
 };
 
-export const generateTypeDefinition = (typeName: string): string => {
-  return `
+export const generateTypeDefinition = (iconTypeName: string, svgDefinitions: SvgDefinition[]): string => {
+  const type = `
     /* ⚠️ Do not edit this file - this file is generated by svg-to-ts*/
     
-    export type ${typeName} = 
-  `;
+    export type ${iconTypeName} = ${svgDefinitions
+    .map(({ typeName }, index) => `'${typeName}'${index === svgDefinitions.length - 1 ? '' : ' | '}`)
+    .join('')};`;
+  const typeObject = `
```

Comment:
This means that currently, we would always generate both, an object for iteration and also a type and there's no option to disable it. I am just thinking of the best way to do this.

I personally would prefer to generate types by default. Then I would also offer the possibility to generate only the const or const and type. Some options like this

````
generateTypes: boolean (default: true);
generateTypeObject:boolean (default: false);
````

Reply:
@felipeplets what do you think?

Reply:
I like the idea. Will push the change later today.